test/Microsoft.OpenApi.Readers.Tests/V2Tests/Samples/twoResponses.json

```diff
@@ -70,13 +71,15 @@
                     }
                 },
                 "produces": [
-                    "application/json"
+                    "application/json",
+                    "application/xml"
                 ]
             }
         }
     },
     "produces": [
-        "application/json"
+        "application/json",
+        "application/xml"
```

Comment:
>       "application/xml" [](start = 1, length = 24)

Sorry I was not clear. Could you remove the `application/xml` from line 82?

Then:

- The check for get and patch should yield the same results (both json and xml)

- The check for post should yield just json.

Reply:
Sure.